src/placeos-rest-api/controllers/api_keys.cr

```diff
@@ -52,7 +52,7 @@ module PlaceOS::Api
     def create
       save_and_respond(Model::ApiKey.from_json(self.body)) do |result|
         @current_api_key = result
-        show
+        render_json(status: :created) { |json| current_api_key.to_public_json(json) }
```

Comment:
What's the reason for the change `to_public_json` here?

Reply:
It's just doing what the show method does but passing in the status since a key was created